### ISSUE TITLE
medic-conf v1.11.1 doesn't convert contact forms correctly

### ISSUE THREAD

**Issue by sglangevin** [medic/medic-conf#57]
When converting contact forms, the group for the place needs to be immediately after the inputs group or the form won't work. The latest version of medic-conf doesn't put the place group in the correct spot resulting in broken forms.

**Comment by alxndrsn** [comment]
Have you got an example form which displays the problem?  If there's an XLS in projects, that would be perfect...

**Comment by sglangevin** [comment]
Try any of the contact forms in the beta folder. Make sure you convert and then upload because I've fixed them manually so that they work correctly.

e.g. https://github.com/medic/medic-projects/blob/master/beta/forms/contact/district_hospital.xlsx

**Comment by alxndrsn** [comment]
Converting them locally does not generate any change against the version in git...

**Comment by sglangevin** [comment]
That's odd - I was definitely having issues with this the other day...let me try again

**Comment by sglangevin** [comment]
After converting over here, I get this change:

![image](https://user-images.githubusercontent.com/1286381/35876089-16c3c7fc-0b26-11e8-9ce3-83ea0f5e3d40.png)

**Comment by alxndrsn** [comment]
Fixed in `1.12.2`.
